plugin/pkg/global/resourcereferencemanager/admission.go

```diff
@@ -771,21 +778,38 @@ func (r *ReferenceManager) ensureBackupBucketReferences(ctx context.Context, old
 	return nil
 }
 
-func (r *ReferenceManager) validateBackupBucketDeletion(ctx context.Context, backupBucketName string) error {
+func (r *ReferenceManager) validateBackupBucketDeleteCollection(ctx context.Context, a admission.Attributes) error {
+	backupBucketList, err := r.gardenCoreClient.Core().BackupBuckets().List(ctx, metav1.ListOptions{LabelSelector: labels.Everything().String()})
+	if err != nil {
+		return err
+	}
+
+	for _, backupBucket := range backupBucketList.Items {
+		if err := r.validateBackupBucketDelete(ctx, utils.NewAttributesWithName(a, backupBucket.Name)); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (r *ReferenceManager) validateBackupBucketDelete(ctx context.Context, a admission.Attributes) error {
```

Comment:
I think the previous name was better, even though I saw that other methods are called `validateDelete`
```suggestion
func (r *ReferenceManager) validateBackupBucketDeletion(ctx context.Context, a admission.Attributes) error {
```